### ISSUE TITLE
server crash

### ISSUE THREAD

**Issue by CoolCuzzy** [fuyutsuki/Texter#135]
```bash
14.02 10:25:56 [Server] INFO [10:25:56.204] [Server thread/INFO]: Enabling Texter v4.1.8
14.02 10:25:56 [Server] INFO [10:25:56.205] [Server thread/CRITICAL]: [Texter] Can't find the pmforms in the virions folder. Please download pmforms and install it correctly.
14.02 10:25:56 [Server] INFO [10:25:56.205] [Server thread/INFO]: Disabling Texter v4.1.8
14.02 10:25:56 [Server] INFO [10:25:56.206] [Server thread/CRITICAL]: Could not enable plugin 'Texter': Plugin disabled itself when attempting to enable it
```
Server crashes to protect integrity

Plugin was working fine previously and all of a sudden i get this error after restarting the server. Could u explain what this means and how to fix if it is from my end?

**Comment by fuyutsuki** [comment]
That error is output when you try to run this plugin with the source and there are not enough dependencies.

However, 4.1.8 does not work properly with Minecraft 1.19.60, so please wait for the next 4.1.9 release.

**Comment by fuyutsuki** [comment]
Hi, just released 4.1.9, please try it.
